acct.bionimbus.org/manifests/scaling/scaling.json

```diff
@@ -0,0 +1,57 @@
+{
+  "arborist": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "dashboard": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "portal": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "fence": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "presigned-url-fence": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "indexd": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "metadata": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "peregrine": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "revproxy": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "sheepdog": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  },
+  "guppy": {
+    "min": 1,
+    "max": 1,
+    "strategy": "auto"
+  }
+}
```

Comment:
this whole block is repetitive to what it's in the manifest I think

Reply:
Should we remove it in the manifest?

Reply:
The redundant config should be harmless.. 
But, ideally, we should tidy up `gitops-qa/qa-acct` and run this automation again.

Reply:
this is only in the PROD manifest. Nothing like this is in the QA env afaik.